rules/windows/process_creation/win_susp_net_execution.yml

```diff
@@ -16,6 +24,11 @@ detection:
         Image:
             - '*\net.exe'
             - '*\net1.exe'
+    filename:
+        OriginalFileName:
+            - 'net.exe'
```

Comment:
better to add net.exe/net1.exe to [special rule](https://github.com/Neo23x0/sigma/blob/master/rules/windows/process_creation/win_renamed_binary.yml) for renamed system files detection